src/bridge/cockpitdbusconfig.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "config.h"
+
+#include "cockpitdbusinternal.h"
+#include "common/cockpitconf.h"
+
+static void
+config_method_call (GDBusConnection *connection,
+                    const gchar *sender,
+                    const gchar *object_path,
+                    const gchar *interface_name,
+                    const gchar *method_name,
+                    GVariant *parameters,
+                    GDBusMethodInvocation *invocation,
+                    gpointer user_data)
+{
+  if (g_str_equal (method_name, "GetString"))
+    {
+      const gchar *section, *key;
+      const char *result;
+
+      g_variant_get (parameters, "(&s&s)", &section, &key);
+
+      /* Ensure we don't work from an old cache, force re-reading the file */
+      cockpit_conf_cleanup ();
```

Comment:
I wouldn't expect the config to be reloaded for every GetString or GetUInt call.  Right now, cockpit-ws never reloads its configuration and I don't think we should change that.  If necessary for the tests, let's introduce a new Reload method.

Reply:
Makes sense, updated to do this. I also fixed the `QUnit.only` leftover, oops!